src/KeyboardControllerViewNativeComponent.ts

```diff
@@ -0,0 +1,23 @@
+import type { HostComponent } from 'react-native';
+// @ts-expect-error - no type definition
+import type { ViewProps } from 'react-native/Libraries/Components/View/ViewPropTypes';
+import type {
+  DirectEventHandler,
+  Float,
+  Int32,
+} from 'react-native/Libraries/Types/CodegenTypes';
+import codegenNativeComponent from 'react-native/Libraries/Utilities/codegenNativeComponent';
+
+type KeyboardMoveEvent = Readonly<{
```

Comment:
I think similar data structure is already defined as `NativeEvent` in `native.ts`. Would it be possible to use one source of truth?

Reply:
Moving type declaration into separate file will cause an error during pods installation 🤷‍♂️